validation-test/Sema/SwiftUI/rdar139237781.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
For SwiftUI tests please use: 

```
// RUN: %target-typecheck-verify-swift -target %target-cpu-apple-macosx10.15 -swift-version 5
// REQUIRES: objc_interop
// REQUIRES: OS=macosx
```

Otherwise it would start failing in some CI configurations we have.

Reply:
Fixed, I don't think it needs -swift-version 5 though